vgc/autopilot.py

```diff
@@ -0,0 +1,54 @@
+"""
+Software interface for the autopilot
+"""
+import time
+from pymavlink import mavutil
```

Comment:
Need to be added to the `requerments.txt`

Reply:
done